### PR TITLE
Adds feature tests for the navigation bar

### PR DESCRIPTION
**Pull request by Ryanspink1**

### CODE REVIEW

File: app/views/partials/nav.erb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <ul class="nav nav-pills custom-nav">
-  <li><a href='/'>Home</a></li>
+  <li id='home'><a href='/'>Home</a></li>
   <li role="presentation" class="dropdown custom-dropdown">
     <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
       Stations <span class="caret"></span>
     </a>
     <ul class="dropdown-menu">
-      <li><a href='/stations'>Stations</a></li>
+      <li id="stations_button"><a href='/stations'>Station Index</a></li>
```
Comment (CjMoore) [inline app/views/partials/nav.erb:8]: Why did you change it from Stations to Station Index?

Comment (Ryanspink1) [reply app/views/partials/nav.erb:8]: It's more descriptive and differentiates itself from the title of the dropdown menu.

Comment (CjMoore) [reply app/views/partials/nav.erb:8]: Ah
